components/n-ui/ads/js/oAdsConfig.js

```diff
@@ -55,20 +55,21 @@ module.exports = function (flags, appName, adOptions) {
 		return zone.join('/');
 	}
 
-	function setViewportMarginBy (variant) {
-		switch (variant) {
-			case '5':
-				return'5%';
-				break;
-			case '10':
-				return '10%';
-				break;
-			case '15':
-				return '15%';
-				break;
-			default:
-				return '0%';
+	function setViewportMarginBy () {
+		let pt = appName;
+		pt = pt.toLowerCase().substr(0, 3);
```

Comment:
Is it necessary to substring it here? Might be more readable later on to refer to 'front-page' (or whatever the full name is)